$NetBSD$

* add -L to be able to link to yaml-cpp before it is installed.

--- test/CMakeLists.txt.orig	2014-02-04 15:13:05.000000000 +0000
+++ test/CMakeLists.txt
@@ -5,6 +5,7 @@ file(GLOB test_new_api_sources new-api/[
 list(APPEND test_sources ${test_new_api_sources})
 
 include_directories(${YAML_CPP_SOURCE_DIR}/test)
+link_directories(${YAML_CPP_SOURCE_DIR})
 
 add_executable(run-tests
 	${test_sources}
